strongbox-data-service/src/main/java/org/carlspring/strongbox/gremlin/server/GremlinServerConfig.java

```diff
@@ -16,7 +18,7 @@
  * @author sbespalov
  */
 @Configuration
-@ConditionalOnProperty(prefix = "strongbox.graph.gremlin.server", name = "enabled", havingValue = "true")
+@ConditionalOnProperty(prefix = "strongbox.graph.gremlin.server", name = ENABLED, havingValue = "true")
```

Comment:
this seems not related to our domain model, so we shouldn't use `Properties` here.

Reply:
Removed in the last commit